Features/Standard/ContentTypeIcon.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Content Type icons
+
+    @javascript
+    Scenario: I can see the generic Content Type icon attached to the Content item name
+        Given a Content Type "custom_content_type" exists
```

Comment:
well, we'll use the Content Type identifier to build a custom CSS class, so here the identifier will be used to make sure this custom class is correctly positioned meaning anybody can replace the generic icon by something else.